aj.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+const (
+	DB_USER     = "njnvxqxp"
+        DB_PASSWORD = "rNuckvNyXEZimXg7u3egwhEXg_eWhRAz"
+	DB_NAME     = "njnvxqxp"
+	DB_HOST     = "rajje.db.elephantsql.com"
+)
+
+var router *gin.Engine
+var db *sqlx.DB
+
+type Patient struct {
+	Id         int    `json:"id" db:"id"`
+	Name       string `json:"name" db:"name"`
+	Illness    string `json:"illness" db:"illness"`
+	Birthdate  string `json:"birth_date" db:"birth_date"`
+	Locationid int    `json:"location_id" db:"location_id"`
+}
+
+type Location struct {
+	Id         int    `json:"id" db:"id"`
+	Name       string `json:"name" db:"name"`
+	Hospitalid int    `json:"hospital_id" db:"hospital_id"`
+}
+
+type Hospital struct {
+	Id               int    `json:"id" db:"id"`
+	Name             string `json:"name" db:"name"`
+	Maxpatientamount int   `json:"max_patient_amount" db:"max_patient_amount"`
+}
+
+func main() {
+	initDatabase()
+	initAPI()
+	err := router.Run(":8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+}
+
+func initDatabase() {
+
+	psqlInfo := fmt.Sprintf("user=%v password=%v dbname=%v host=%v sslmode=disable", DB_USER, DB_PASSWORD, DB_NAME, DB_HOST)
+	var err error
+
+	db, err = sqlx.Open("postgres", psqlInfo)

```

Comment:
You can use `sqlx.MustConnect` to decrease lines of code.

Reply:
Ok